prisma/schema.prisma

```diff
@@ -12,12 +12,79 @@ datasource db {
 }
 
 model User {
-  id        String   @id @default(uuid())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  id            String        @id @default(uuid())
+  email         String        @unique
+  firstName     String?
+  lastName      String?
+  birthday      String?
+  imageUrl      String?
+  createdAt     DateTime      @default(now())
+  updatedAt     DateTime      @updatedAt
+  clerkId       String        @unique
+  financeTeams  FinanceTeam[] @relation("UserFinanceTeams")
+  items         Item[]
+  roles         UserRole[]
+}
+
+model FinanceTeam {
```

Comment:
One of the concepts of the app is to allow couples to budget together. I created a scalable/flexible data model that allows for a "Finance Team" where a user can be part of multiple finance teams.